server/spinwick.go

```diff
@@ -606,3 +625,21 @@ func (s *Server) isSpinWickLabelInLabels(labels []string) bool {
 
 	return false
 }
+
+func (s *Server) removeServerUpdateOldComments(comments []*github.IssueComment, serverMessages []string, pr *model.PullRequest) {
```

Comment:
This function name is a little confusing to me. Maybe something like `removeCommentsWithSpecificMessages`?

Reply:
sounds good